src/internal/m365/onedrive/collection.go

```diff
@@ -531,11 +534,29 @@ func (oc *Collection) populateDriveItem(
 		// attempts to read bytes.  Assumption is that kopia will check things
 		// like file modtimes before attempting to read.
 		itemReader := lazy.NewLazyReadCloser(func() (io.ReadCloser, error) {
-			itemData, err := oc.getDriveItemContent(ctx, oc.driveID, item, errs)
+			rc, err := oc.getDriveItemContent(ctx, oc.driveID, item, errs)
 			if err != nil {
 				return nil, err
 			}
 
+			itemData := rc
+
+			// Add per item extensions if available
+			if len(itemExtensionFactory) > 0 {
+				extRc, extInfo, err := extensions.AddItemExtensions(
+					ctx,
+					rc,
+					itemInfo,
+					itemExtensionFactory)
+				if err != nil {
+					el.AddRecoverable(ctx, clues.Wrap(err, "adding extensions").Label(fault.LabelForceNoBackupCreation))
+					return nil, clues.Stack(err).WithClues(ctx)
```

Comment:
Shouldn't need `WithClues(ctx)` here, since the error producer accepted the `ctx` already.  This could unintentionally overwrite some downstream values.